### DISCUSSION TITLE
Active TAB URL - How to save it in the table?

### DISCUSSION THREAD

**Discussion by alpha-code2019** [AutomaApp/automa#524] [Q&A] [answered]
I tried and tried and tried,

But the only solution I came up with was the use of the "Clipboard" block,
which is not an elegant solution at all.

**I have to click and copy the URL in the browser address bar, then Automa adds it to the table.**

Isn't there a more elegant approach/solution?

Thank you!

**Comment by Kholid060** [comment, marked as answer]
Use the javascript block,

```js
const currentUrl = window.location.href;
automaNextBlock({ columnName: currentUrl });
```

**Comment by alpha-code2019** [comment]
Thank you, Ahmad - it works like a Swiss clockwork!

Am So., 5. Juni 2022 um 04:08 Uhr schrieb Ahmad Kholid <
***@***.***>:
